CONTRIBUTING.md

```diff
@@ -53,3 +53,14 @@ following:
             personal information I submit with it, including my sign-off) is
             maintained indefinitely and may be redistributed consistent with
             this project or the open source license(s) involved.
+
+## Maintainers: Create Releases
+
+The creation of public releases is a partially automated process:
+
+1. update code and create release tags: `VERSION=<x.x.x> make release`
+2. push to GitHub: `git push origin main && git push origin v<x.x.x>`
+3. wait for release action to finish (public release is created)
+4. download signed xpi artifact
+5. update release information in `.pages/firefox/updates.json`
+6. commit updated release information to master
```

Comment:
We were discussing if you cannot make the workflow wait for the release to be done, fetch and patch the updates.json automatically. Any blockers for this?

Reply:
As the wait-for-sign is now implemented, there are no further blockers. But I don't really like the idea that the CI pushes code to the repo. I'll think about it again.

Reply:
Finally, we can fully-automatically release new version of the FF extension.